assume/scenario/loader_csv.py

```diff
@@ -367,7 +368,7 @@ async def load_scenario_folder_async(
     if not learning_config.get("trained_policies_save_path"):
         learning_config[
             "trained_policies_save_path"
-        ] = f"./learned_strategies/{study_case}"
+        ] = f"./learned_strategies/{study_case}/last_policies"
```

Comment:
last_policies are set twice now, here and also in load functions of the learning strategies.